### PR TITLE
init-rubocop

### PR DESCRIPTION
**Pull request by datcnt-0874**
Screenshot from 2019-04-04 01-08-01

### CODE REVIEW

**Comment by datcnt-0874** [comment]
ready

File: Rakefile [hunk 1]
```diff
@@ -1,6 +1,5 @@
 # Add your own tasks in files placed in lib/tasks ending in .rake,
-# for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
-
-require_relative 'config/application'
+# for example lib/tasks/capistrano.rake, and they will automatically
+require_relative "config/application"
```
Comment (DuongNT1512) [inline Rakefile]: single quote

File: config.ru [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # This file is used by Rack-based servers to start the application.
 
-require_relative 'config/environment'
+require_relative "config/environment"
```
Comment (DuongNT1512) [inline config.ru]: single quote

**Comment by datcnt-0874** [comment]
ready

**Comment by datcnt-0874** [comment]
ready

**Pull request merged**